cmd/imageproxy/main.go

```diff
@@ -112,6 +102,29 @@ func main() {
 	log.Fatal(http.ListenAndServe(*addr, nil))
 }
 
+type SignatureKeyList struct {
```

Comment:
This type doesn't need to be exported.  There's not a ton of harm, given that it's in a main package, but still worth fixing.  Maybe rename the var above to `signatureKeys` to avoid the name conflict?

I suspect this also doesn't need to be a struct.  Could just be:

``` go
type signatureKeyList [][]byte
```

I think that should work?